mipengine/algorithms/anova_oneway.py

```diff
@@ -41,13 +41,18 @@ def run(self):
             keyword_args=dict(y=Y_relation, x=X_relation, covar_enums=covar_enums),
             share_to_global=[True, True],
         )
-
-        result = global_run(
-            func=global1,
-            keyword_args=dict(
-                sec_local_transfer=sec_local_transfer, local_transfers=local_transfers
-            ),
-        )
+        try:
+            result = global_run(
+                func=global1,
+                keyword_args=dict(
+                    sec_local_transfer=sec_local_transfer,
+                    local_transfers=local_transfers,
+                ),
+            )
+        except Exception as ex:
+            if "Cannot perform Anova when there is only one level." in str(ex):
```

Comment:
Please add this line here:
`# TODO https://team-1617704806227.atlassian.net/browse/MIP-682`

This is the ticket about proper error handling.